### PR TITLE
Added in my characters

### PR DESCRIPTION
**Pull request by TheProgrammingProject**
Created new branch because I didn't want to remove any new chars not in the downloadable version and I was unsure if I had accidently duplicated chars

### CODE REVIEW

File: WtWSMS_EBS/history/characters/roman.txt [hunk 1]
```diff
@@ -1210,7 +1210,13 @@
 		death="290.1.1"
 	}
 }
-
+145250 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: ID duplicate with a WtWSMS character, which is Aurelianus too.

File: WtWSMS_EBS/history/characters/roman.txt [hunk 2]
```diff
@@ -1224,7 +1230,23 @@
 		death="240.1.1"
 	}
 }
-
+145252 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: ID duplicate with a WtWSMS character, which is Claudius too.

File: WtWSMS_EBS/history/characters/roman.txt [hunk 3]
```diff
@@ -1224,7 +1230,23 @@
...
+	name = "Claudius"
+	dynasty = 10000000003
+	religion = "hellenic_pagan"
+	culture = "roman"
+	trait = "tough_soldier"
+	214.5.10 = { birth = yes }
+	270.1.15 = { death = yes }
+}
+145253 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: ID duplicate with a WtWSMS character, which is Quintillus too.

File: WtWSMS_EBS/history/characters/roman.txt [hunk 4]
```diff
@@ -1286,7 +1308,15 @@
 		death="253.7.1"
 	}
 }
-
+145258 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: ID duplicate with a WtWSMS character, which is Decius too.

File: WtWSMS_EBS/history/characters/roman.txt [hunk 5]
```diff
@@ -4161,8 +4190,8 @@
 	}
 }
 1300020 = {
-	name="Helpidius" #PLRE vol.2 Helpidius 1
-	#?consularis Campaniae a. 396, ?proconsul Africae 402
+	name="Helpidius" PLRE vol.2 Helpidius 1
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: Comments don't work without `#`, and there's no need for spaces in the end of the line.

File: WtWSMS_EBS/history/characters/roman.txt [hunk 6]
```diff
@@ -4904,12 +4933,6 @@
 	470.1.1={
 		birth=yes
 	}
-	516.1.1 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: Duces were military commanders, hence the commander minor title given to him.

File: WtWSMS_EBS/history/characters/roman.txt [hunk 7]
```diff
@@ -5823,8 +5846,8 @@
 	religion = nicene
 	culture = roman
 	trait = zealous
-	430.1.1 = { birth = yes }
-	490.1.1 = { death = yes }
+	410.1.1 = { birth = yes }
+	480.1.1 = { death = yes }
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: ?

File: WtWSMS_EBS/history/characters/roman.txt [hunk 8]
```diff
@@ -5407,8 +5430,8 @@
 	# governor of Sicily 400
 	religion = nicene
 	culture = roman
-	380.1.1 = { birth = yes }
-	439.1.1 = { death = yes }
+	350.1.1 = { birth = yes }
+	410.1.1 = { death = yes }
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: ?

**Review by IhateTrains — CHANGES_REQUESTED** [review]

File: WtWSMS_EBS/history/characters/roman.txt [hunk 9]
```diff
@@ -4081,7 +4111,6 @@
 }
 1300013 = {
 	name="Heliodorus" #PLRE vol.2 Fl. Heliodorus 9
-	#?provincial governor, later proconsul of Asia
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt]: ?

File: WtWSMS_EBS/history/characters/roman.txt (hunk 2 above)
Comment (TheProgrammingProject) [reply WtWSMS_EBS/history/characters/roman.txt]: fixed

File: WtWSMS_EBS/history/characters/roman.txt [hunk 10]
```diff
@@ -5904,4 +5901,88 @@
...
+145242 = {
+	name = "Carus"
+	dynasty = 10000000004
+	religion = "hellenic_pagan"
+	culture = "roman"
+	trait = "inspiring_leader"
+	222.4.5 = { birth = yes }
+	283.8.1 = { death = yes }
+}
+145243 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt:6304]: ID duplicate with a WtWSMS character, which is Numerianus too.

File: WtWSMS_EBS/history/characters/roman.txt [hunk 11]
```diff
@@ -5904,4 +5901,88 @@
...
+}
+1400006 = {
+	name = "Volusianus"
+	dynasty = 1022208
+	religion = "hellenic_pagan"
+	culture = "roman"
+	230.1.1 = { birth = yes }
+	253.8.1 = { death = yes }
+}
+145242 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt:6295]: ID duplicate with a WtWSMS character, which is Carus too.

**Comment by IhateTrains** [comment]
Instead of adding duplicate characters, you should just edit the existing versions present in EBS or WtWSMS (EBS only has the files that were modified in comparison with WtWSMS, so you might need to find the characters directly in WtWSMS).

File: WtWSMS_EBS/history/characters/roman.txt [hunk 12]
```diff
@@ -5904,4 +5901,88 @@
...
+}
+145243 = {
+	name = "Numerian"
+	dynasty = 10000000004
+	religion = "hellenic_pagan"
+	culture = "roman"
+	251.1.1 = { birth = yes } #Conjectural
+	284.11.20 = { death = yes }
+}
+145244 = {
```
Comment (IhateTrains) [inline WtWSMS_EBS/history/characters/roman.txt:6312]: ID duplicate with a WtWSMS character, which is Carinus too.

**Comment by TheProgrammingProject** [comment]
I can't figure out what the problem between this and the main branch is.

**Comment by IhateTrains** [comment]
I've resolved the conflicts.
There are still some changes that need to be done, see the unresolved conversations concerning duplicate IDs.

**Comment by TheProgrammingProject** [comment]
Ok, thanks. Can I close the pull request now?

**Comment by IhateTrains** [comment]
Sure, but why?

**Comment by TheProgrammingProject** [comment]
Having an open pull request makes it look like it wasn’t merged back in…

**Comment by IhateTrains** [comment]
Sorry, seems that my commits have confused you. I merged the master branch into your branch to resolve the conflicts and make the pull request mergeable. The changes from this pull request haven't been merged into the master branch yet.

**Pull request closed without merging**